File: rb/test/exists_test.rb
# IamSmart SDK exists test

require "minitest/autorun"
require_relative "../IamSmart_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = IamSmartSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- IamSmart SDK exists test

local sdk = require("iam-smart_sdk")

describe("IamSmartSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
